docs/install.qmd

```diff
@@ -52,23 +55,28 @@ One can also use Ribasim Python to build entire models from base data, such that
 
 # Download {#sec-download}
 
-- Ribasim executable - Windows: [ribasim_windows.zip](https://github.com/Deltares/Ribasim/releases/latest/download/ribasim_windows.zip)
-- Ribasim executable - Linux: [ribasim_linux.zip](https://github.com/Deltares/Ribasim/releases/latest/download/ribasim_linux.zip)
-- QGIS plugin: [ribasim_qgis.zip](https://github.com/Deltares/Ribasim/releases/latest/download/ribasim_qgis.zip).
-- Generated testmodels: [generated_testmodels.zip](https://github.com/Deltares/Ribasim/releases/latest/download/generated_testmodels.zip)
-
-The Ribasim Python package is [registered in PyPI](https://pypi.org/project/ribasim/) and [conda-forge](https://prefix.dev/channels/conda-forge/packages/ribasim) and can therefore be installed with [pip](https://docs.python.org/3/installing/index.html), [conda](https://docs.conda.io/) or [pixi](https://pixi.sh/):
-```
-pip install ribasim
-```
+There are three main components of the Ribasim software package.
+They are the Ribasim core (written in Julia language), the Ribasim Python package and the Ribasim QGIS plugin.
+The Ribasim core is the computation kernel of the Ribasim software package and is used to run simulations.
+The Ribasim Python package is used to build Ribasim models programmatically with Python script.
+The Ribasim QGIS plugin is used to construct or view a model in GUI.
+Users can choose to use Ribasim Python or QGIS plugin or a combination of them to build a Ribasim model.
+And then use the Ribasim core to run the simulation.
 
```

Comment:
```suggestion
```

Explaining the components is done above in the Components section. If those are not clear enough it is best to improve that section rather than repeating the same message here.